turbo/execution/eth1/forkchoice.go

```diff
@@ -295,12 +294,6 @@ func (e *EthereumExecutionModule) updateForkChoice(ctx context.Context, original
 			sendForkchoiceErrorWithoutWaiting(outcomeCh, err)
 			return
 		}
-		if unwindToGenesis {
```

Comment:
Seems not related to this PR. Let's move it to own PR?

Reply:
It is related, now we do not need this hack anymore